kbase-extension/static/kbase/js/common/jobManager.js

```diff
@@ -302,6 +302,31 @@ define(['common/html', 'common/jobs', 'common/ui', 'util/string'], (html, Jobs,
             this.runUpdateHandlers(jobArray);
             return this.model;
         }
+
+        initBatchJob(message) {
```

Comment:
this will be fleshed out more; for now, it just sticks the child jobs into the saved jobs array.